package.json

```diff
@@ -35,6 +36,7 @@
   },
   "dependencies": {
     "@types/bitcore-lib": "0.15.1",
+    "@types/node-fetch": "1.6.9",
```

Comment:
I think they are needed as normal dependencies due to Po.et JS being a library, not 100% sure now though. Will try to check that later today — feel free to leave it out of this PR to get it merged faster though.

Reply:
Check out this discussion: https://github.com/Microsoft/types-publisher/issues/81

This is what persuaded me to move them into `dependencies`.

Reply:
I've gone through the thread again and looked at a couple of other discussions on this topic and the consensus is that it should be in dependencies if the package is going to be a library. See here for another example:
https://github.com/apollographql/apollo-client/issues/713

I'm pretty convinced this is the right path.

Reply:
I don't think the size of the @type packages is an issue because they are each 16kb uncompressed, so hardly worth worrying about.